src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -167,7 +167,28 @@ namespace winrt::TerminalApp::implementation
         _newTabButton.Click([weakThis{ get_weak() }](auto&&, auto&&) {
             if (auto page{ weakThis.get() })
             {
-                page->_OpenNewTab(nullptr);
+                // if alt is pressed, open a pane
+                const CoreWindow window = CoreWindow::GetForCurrentThread();
+                const auto rAltState = window.GetKeyState(VirtualKey::RightMenu);
+                const auto lAltState = window.GetKeyState(VirtualKey::LeftMenu);
+                const bool altPressed = WI_IsFlagSet(lAltState, CoreVirtualKeyStates::Down) ||
+                                        WI_IsFlagSet(rAltState, CoreVirtualKeyStates::Down);
+
```

Comment:
What if it's Ctrl+Alt, Shift+Alt, or Ctrl+Shift+Alt? Do we want to exclude those?

Reply:
Currently its implemented the same way as if you want to split pane from the dropdown (the down-arrow button and then selecting a profile) - I imagine it would be best if we keep both of them as similar as possible? I.e. since Ctrl+Alt etc are not excluded in the split pane via dropdown implementation I don't think they should be excluded here either

Reply:
OK. Agreed.